pyqtgraph/graphicsItems/PlotCurveItem.py

```diff
@@ -416,53 +421,56 @@ def setSkipFiniteCheck(self, skipFiniteCheck):
                         high-contrast border. Any single argument accepted by
                         :func:`mkPen <pyqtgraph.mkPen>` is allowed.
-        fillLevel       (float or None) Fill the area 'under' the curve to
-                        *fillLevel*
-        fillOutline     (bool) If True, an outline surrounding the *fillLevel*
+        fillLevel       (float or None) Fill the area under the curve to
+                        the specified value.
+        fillOutline     (bool) If True, an outline surrounding the `fillLevel`
                         area is drawn.
-        brush           QBrush to use when filling. Any single argument accepted
+        brush           Brush to use when filling. Any single argument accepted
                         by :func:`mkBrush <pyqtgraph.mkBrush>` is allowed.
         antialias       (bool) Whether to use antialiasing when drawing. This
                         is disabled by default because it decreases performance.
-        stepMode        (str or None) If "center", a step is drawn using the x
-                        values as boundaries and the given y values are
+        stepMode        (str or None) If 'center', a step is drawn using the `x`
+                        values as boundaries and the given `y` values are
                         associated to the mid-points between the boundaries of
                         each step. This is commonly used when drawing
-                        histograms. Note that in this case, len(x) == len(y) + 1
-                        If "left" or "right", the step is drawn assuming that
-                        the y value is associated to the left or right boundary,
-                        respectively. In this case len(x) == len(y)
-                        If not passed or an empty string or None is passed, the
+                        histograms. Note that in this case, ``len(x) == len(y) + 1``
+                        
+                        If 'left' or 'right', the step is drawn assuming that
+                        the `y` value is associated to the left or right boundary,
+                        respectively. In this case ``len(x) == len(y)``
+                        If not passed or an empty string or `None` is passed, the
                         step mode is not enabled.
         connect         Argument specifying how vertexes should be connected
-                        by line segments. Default is "all", indicating full
-                        connection. "pairs" causes only even-numbered segments
-                        to be drawn. "finite" causes segments to be omitted if
-                        they are attached to nan or inf values. For any other
-                        connectivity, specify an array of boolean values.
+                        by line segments. 
+                        
+                            | 'all' (default) indicates full connection. 
+                            | 'pairs' omits even-numbered segments. 
```

Comment:
The previous docs say that even-numbered segments are drawn, presumably counting from zero.

Reply:
Hmm. If you naturally start counting from zero, then neither of these descriptions seems particularly helpful.

"  'pairs' draws one separate line segment for each two points given, starting with the first point.   " ?

Reply:
I suppose ``` 'pairs' draws one separate line segment for each two points given.``` would be good enough. Don't think there is any doubt about it starting from the first point.

Reply:
Agreed, and changed.